src/client_side_reply.cc

```diff
@@ -1129,6 +1131,40 @@ clientReplyContext::storeNotOKTransferDone() const
     }
 }
 
+/* A write has completed, what is the next status based on the
+ * canonical request data?
+ * 1 something is wrong
+ * 0 nothing is wrong.
+ *
+ */
+int
+clientHttpRequestStatus(int fd, ClientHttpRequest const *http)
+{
+#if SIZEOF_INT64_T == 4
```

Comment:
Please do not restore dead code that you removed in 5352173435912e825caded3d41578f953dcc9ca6.